charts/spv-wallet-stack/charts/spv-wallet/templates/configmap.yml

```diff
@@ -8,91 +8,110 @@ data:
+        debug: {{ .Values.config.db.sqlite.debug }}
+        max_connection_idle_time: {{ .Values.config.db.sqlite.max_connection_idle_time }}
+        max_connection_time: {{ .Values.config.db.sqlite.max_connection_time }}
+        max_idle_connections: {{ .Values.config.db.sqlite.max_idle_connections }}
+        max_open_connections: {{ .Values.config.db.sqlite.max_open_connections }}
+        shared: {{ .Values.config.db.sqlite.shared }}
+    debug_profiling: {{ .Values.config.debug_profiling }}
+    disable_itc: {{ .Values.config.disable_itc }}
+    arc:
+      url: {{ .Values.config.arc.url }}
+      token: {{ .Values.config.arc.token }}
+      {{- if .Values.config.nodes.deployment_id }}
+        deployment_id: {{ .Values.config.nodes.deployment_id }}
+      {{- else }}
+        deployment_id: 'spv-wallet-{{ printf "%s-$s"  (include "spv-wallet.fullname" .) .Values.global.domainName | adler32sum }}'
+      {{- end }}
       callback:
-        callback_host: 'https://{{.Values.global.domainName}}'
-      use_fee_quotes: {{ .Values.config.nodes.use_fee_quotes }}
+        enabled: {{ .Values.config.arc.callback.enabled }}
+        host: 'https://{{.Values.global.domainName}}'
+        if (.Values.config.arc.callback.token) {  
+          token: .Values.config.arc.callback.token  
+        } 
+      use_fee_quotes: {{ .Values.config.arc.use_fee_quotes }}
       fee_unit:
-        satoshis: {{ .Values.config.nodes.fee_unit.satoshis }}
-        bytes: {{ .Values.config.nodes.fee_unit.bytes }}
+        satoshis: {{ .Values.config.arc.fee_unit.satoshis }}
+        bytes: {{ .Values.config.arc.fee_unit.bytes }}
     notifications:
       enabled: {{ .Values.config.notifications.enabled }}
-      webhook_endpoint: "{{ .Values.config.notifications.webhook_endpoint }}"
+    block_headers_service:
+      auth_token: ""
+      {{- $defaultUrl := printf "http://%s-block-headers-service" .Release.Name | trunc 63 | trimSuffix "-" -}}
+      {{- $blockHeadersServiceUrl := default $defaultUrl .Values.config.block_headers_service.url -}}
+      url: {{ $blockHeadersServiceUrl }}
+
     paymail:
       beef:
-        {{- if .Values.config.paymail.beef.block_headers_service_url }}
-        block_headers_service_url: {{ .Values.config.paymail.beef.block_headers_service_url }}
-        {{- else }}
-        block_headers_service_url: 'http://{{- printf "%s-block-headers-service" .Release.Name | trunc 63 | trimSuffix "-" }}/api/v1/chain/merkleroot/verify'
-        {{- end }}
-        use_beef: {{  .Values.config.paymail.beef.use_beef }}
+        block_headers_service_auth_token: ""
+        {{- $defaultUrl := printf "http://%s-block-headers-service/api/v1/chain/merkleroot/verify" .Release.Name | trunc 63 | trimSuffix "-" -}}
+        {{- $blockHeadersServiceUrl := printf "%s/api/v1/chain/merkleroot/verify" .Values.config.block_headers_service.url | default $defaultUrl -}}
+        url: {{ $blockHeadersServiceUrl }}
```

Comment:
I think that the property name should be `block_headers_service_url` not `url` in line 91